src/components/button-toggle/button-toggle.ts

```diff
@@ -1,26 +1,42 @@
 import {
-  Component,
-  ContentChildren,
-  Directive,
-  EventEmitter,
-  HostBinding,
-  Input,
-  OnInit,
-  Optional,
-  Output,
-  QueryList,
-  ViewEncapsulation,
-  forwardRef
+    Component,
+    ContentChildren,
+    Directive,
+    EventEmitter,
+    HostBinding,
+    Input,
+    OnInit,
+    Optional,
+    Output,
+    Provider,
+    QueryList,
+    ViewEncapsulation,
+    forwardRef
 } from '@angular/core';
+import {
+    NG_VALUE_ACCESSOR,
+    ControlValueAccessor,
+} from '@angular/common';
 import {Observable} from 'rxjs/Observable';
 import {
-  MdUniqueSelectionDispatcher
+    MdUniqueSelectionDispatcher
 } from '@angular2-material/core/coordination/unique-selection-dispatcher';
 import {BooleanFieldValue} from '@angular2-material/core/annotations/field-value';
 
 export type ToggleType = 'checkbox' | 'radio';
 
 
+
+/**
+ * Provider Expression that allows md-button-toggle-group to register as a ControlValueAccessor.
+ * This allows it to support [(ngModel)] and ngControl.
+ */
+const MD_BUTTON_TOGGLE_GROUP_CONTROL_VALUE_ACCESSOR = new Provider(
```

Comment:
Nitpick: any chance you could make this a bit shorter?  Maybe lose the CONTROL?  MD_BUTTON_TOGGLE_GROUP_VALUE_ACCESSOR